src/sonic-config-engine/minigraph.py

```diff
@@ -66,11 +66,11 @@ def get_peer_switch_info(link_metadata, devices):
     for data in link_metadata.values():
         if "PeerSwitch" in data:
             peer_hostname = data["PeerSwitch"]
-            peer_lo_addr = devices[peer_hostname]["lo_addr"] 
-            peer_lo_addr = ipaddress.ip_network(UNICODE_TYPE(peer_lo_addr))
+            peer_lo_addr_str = devices[peer_hostname]["lo_addr"]
+            peer_lo_addr = ipaddress.ip_network(UNICODE_TYPE(peer_lo_addr_str)) if peer_lo_addr_str else None
```

Comment:
@yxieca , not sure I understood.. if "PeerSwitch" is defined, why should `lo` be None? Is it some sort of missing data?

Reply:
could be. Lawrence is not here to discuss. I think he didn't update mingiraph template to include the peer lo address information.